src/tqec/constructions/qubit.py

```diff
@@ -44,10 +44,10 @@ def __init__(self, dim: int) -> None:
 
 
 class ScalableQubitRectangle(TemplateOrchestrator):
-    def __init__(self, width: int, height: int) -> None:
+    def __init__(self, width: int, height: int, scale_width: bool | None = None) -> None:
```

Comment:
I would copy-paste the documentation of `ScalableRectangle` here also, to explain clearly what `width`, `height` and more importantly `scale_width` are doing.